code/modules/power/cables/cable.dm

```diff
@@ -35,9 +35,9 @@ By design, d1 is the smallest direction and d2 is the highest
 	layer = LOW_OBJ_LAYER //isset to WIRE_LAYER when spawned
 
 	/// The direction of endpoint one of this cable
-	var/d1 = 0
+	var/tmp/d1 = 0
```

Comment:
I don't understand /tmp variables, what's the benefit to changing their type here?

Reply:
discussed offline